docs/communication_protocols/timer_based_protocol.rst

```diff
@@ -0,0 +1,147 @@
+
+**Vertiq modules support DSHOT, but they do not support Bidirectional DSHOT.** See the linked article from Chris Landa above for more details on the difference between the two protocols. 
+
+DSHOT has various different speeds, currently Vertiq modules support:
+
+* DSHOT150
+* DSHOT300
+* DSHOT600
+* DSHOT1200
+
+.. _hobby_other_protocols:
+
+Additional Protocols
+*********************
+Vertiq modules also support several other, less commonly used, timer based protocols:
+
+* OneShot125
+* OneShot42
+* Multishot
+
+Similarly to `Standard PWM`_, the commands sent by these protocols are interpreted by a Vertiq module as encoding a number between 0.0 to 1.0. Exactly how Vertiq modules convert 
+this number into a throttle command depends on the configurations of the module. Refer to the :ref:`throttle_mapping` section for more details on that mapping for speed modules and 
+the :ref:`throttle_def` section for more details on how a speed module defines a throttle command.
+
+.. _timer_based_v_serial_flowchart:
+
+Interaction with Serial Communication
+======================================
+The connector used for inputting timer based protocols on Vertiq modules is also used for serial communication. Serial communication is used for connecting to the 
+modules with IQ Control Center or interacting with them using the Vertiq APIs. **Because the connector is shared between timer based protocols and serial communication, only one of them can be active at any time.** On startup, the module will 
+look for valid messages of either the currently configured timer based protocol or the Vertiq serial protocol :ref:`IQUART <uart_messaging>`. **When it detects the first valid message of 
+either type, it will lock-on to that type of communication, and stop listening for any other types of messages on that physical interface. This behavior is slightly different 
+when using the** *Autodetect* **setting for the** *Communication* **parameter**. See more :ref:`below <the_communication_parameter>`.
+
+For example, if after startup a DSHOT message is the first thing sent to the module, and the module is set to use *Autodetect*, it will then only listen for 
+additional DSHOT messages, ignoring any serial communication (as well as all other timer based protocols). The module must be rebooted before you can switch which protocol it will listen to. **This means that you cannot connect to 
+the IQ Control Center after controlling the module with a timer based protocol unless you reboot, and vice versa.** This behavior is summarized in the diagram below.
+
+Also, note that for modules that support DroneCAN, DroneCAN uses a separate physical interface from timer based protocols and serial communication, so DroneCAN communication can still be used when using timer based protocols.
+
+.. image:: ../_static/tutorial_images/pwm_flight_controller/comms_flow.png
```

Comment:
One more annoying place that the word hobby protocols still exists.....inside this flow chart